Generator/FCBBarCodeGenerator.swift

```diff
@@ -26,25 +26,25 @@ struct FCBBarCodeGenerator {
     
     // MARK: Private Methods
     
-    private func imageWithFilter(filter : CIFilter, size: CGSize) -> UIImage? {
+    fileprivate func imageWithFilter(_ filter : CIFilter, size: CGSize) -> UIImage? {
```

Comment:
@ezefranca same thing here, maybe
`fileprivate func image(with filter : CIFilter, size: CGSize) -> UIImage?`
would look better on swift 3, what do you think?